### ISSUE TITLE
Conda-forge version of slycot 0.3.3 generates errors

### ISSUE THREAD

**Issue by murrayrm** [python-control/python-control#217]
When installing `slycot` from conda-forge (v0.3.3), I get the following error when running `python setup.py test`:
```
FAIL: testMinrealBrute (control.tests.minreal_test.TestMinreal)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/murray/Dropbox/macosx/src/python-control/murrayrm/control/tests/minreal_test.py", line 62, in testMinrealBrute
    raise e
  File "/Users/murray/Dropbox/macosx/src/python-control/murrayrm/control/tests/minreal_test.py", line 57, in testMinrealBrute
    ht1.den[0][0], ht2.den[0][0])
  File "/Users/murray/Dropbox/macosx/src/python-control/murrayrm/control/tests/minreal_test.py", line 35, in assert_numden_almost_equal
    np.testing.assert_array_almost_equal(n1, n2)
  File "/Users/murray/anaconda/envs/python3.6-oldslycot/lib/python3.6/site-packages/numpy/testing/nose_tools/utils.py", line 963, in assert_array_almost_equal
    precision=decimal)
  File "/Users/murray/anaconda/envs/python3.6-oldslycot/lib/python3.6/site-packages/numpy/testing/nose_tools/utils.py", line 717, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 6 decimals

(shapes (3,), (4,) mismatch)
 x: array([ -1.33214 , -10.368378, -13.703545])
 y: array([ -1.33214 , -13.297947, -36.505109, -30.136079])
```
This error does not occur when using pip version of `slycot`.  Need to understand what is going on before releasing 0.8.0.

Note: conda-forge version of `slycot` uses OpenBLAS, pip version uses `lapack` (I think).

**Comment by moorepants** [comment]
This error occurs with conda forge's slycot 0.2.0, 0.3.0, and 0.3.3.

**Comment by moorepants** [comment]
How precise is this test? I can imagine minreal being finicky because of floating point operations. How robust is it to different platforms, architectures, and blas/lapack implementations?

**Comment by murrayrm** [comment]
Fixed in #220.
